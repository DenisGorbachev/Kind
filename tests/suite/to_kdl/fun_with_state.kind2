#kdl_state = MyFn.state
MyFn : U60 {
  1
}

MyFn.state : U60 {
  2
}