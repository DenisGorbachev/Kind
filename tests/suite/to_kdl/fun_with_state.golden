// MyFn : U60
fun (MyFn) {
  (MyFn) =
    #1
} with {
    #2
}

